src/BaGet.Core/Extensions/DependencyInjectionExtensions.cs

```diff
@@ -133,6 +134,11 @@ private static void AddDefaultProviders(this IServiceCollection services)
                     return provider.GetRequiredService<FileStorageService>();
                 }
 
+                if (configuration.HasStorageType("mysql"))
```

Comment:
I realized this should perhaps be "database" not "mysql".  What do you think @loic-sharma?